src/contracts/security/security.service.ts

```diff
@@ -45,16 +47,59 @@ export class SecurityService {
     return contractWithSigner;
   }
 
-  /**
-   * Returns the maximum number of deposits per transaction from the contract
-   */
-  public async getMaxDeposits(blockTag?: BlockTag): Promise<number> {
+  public async getContractV2WithSigner() {
```

Comment:
That's very strange, that contract V2 is first version of dsm contract.
Also it has trimmed abi and this is not pointed in function name.

To make code cleaner I propose
- Name contract's abis with names that used in Lido everywhere (security -> deposit security module)
- Method getContractV2WithSigner should return next version of contract
- Maybe rename to getDepositSecurityModuleContractV2?
- Seems SecurityService do a lot of staff. Maybe create base class that contains common methods and use it in DepositService, UnvettingService and PauseService.

Reply:
Seems choosing abi should be done in repositoryService

Reply:
will think about fix